src/Application/Common/Exceptions/ConflictException.cs

```diff
@@ -0,0 +1,8 @@
+namespace Application.Common.Exceptions;
+
+public class ConflictException : Exception
+{
+    public ConflictException(string message) : base(message)
+    {
+    }
+}
```

Comment:
what type of exception that we listed as ConfilctException?

Reply:
we try to update/add a resource which has a key constraint and that resource violates it